scm/github.go

```diff
@@ -266,15 +266,18 @@ func (s *GithubSCM) CreateGroup(ctx context.Context, opt *GroupOptions) (*Reposi
 	if !opt.valid() {
 		return nil, E(op, m, fmt.Errorf("missing fields: %+v", *opt))
 	}
+	slug.Lowercase = false
+	groupName := slug.Make(opt.GroupName)
```

Comment:
I don't want to change the group name entered by students in the frontend. We should prevent students from creating a group name that isn't a valid repository name. We should do it in the frontend.